File: littleSubarraySum.c
#include <stdio.h>

int main(){
    int N,A,B;
    scanf("%d %d %d", &N, &A, &B);

    int arr[100];

    for (int i =0; i<N; i++){
        scanf("%d", &arr[i]);
    }

    int sum = 0;
    for(int i = A; i<=B; i++){
        sum += arr[i];
    }

    printf("%d\n", sum);

    return 0;
}

File: littleSubarraySum.swift
import Foundation

let input = readLine()!.split(separator: " ").map{Int($0)!}

let N = input[0]
let A = input[1]
let B = input[2]

let arr = readLine()!.split(separator: " ").map{Int($0)!}

var sum = 0;
for i in A...B{
    sum += arr[i]
}

print(sum)